### PR TITLE
feat: Add memorable icons to the OM data governance guide

### PR DESCRIPTION
**Pull request by JimHagan**
## Give us some context

* Added a TechTileGrid on intro page to use graphic icons to navigate to the sections of the guide.
* Put marker icons to denote specific sections of the guide.

image

image

### CODE REVIEW

**Comment by JimHagan** [comment]
@urbiz-nr Not sure what the build fail was?  Is it at all related to the images I added?

**Comment by urbiz-nr** [comment]
I'll have a look asap, @JimHagan! Is this very urgent?

**Comment by urbiz-nr** [comment]
Will be ready by then, no problem!

File: src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-coe.mdx [hunk 1]
```diff
@@ -18,18 +18,27 @@ redirects:
   - /telemetry-data-platform/get-started/manage-data/data-governance
 ---
 
+import rolespracticesicon from './images/oma-oe-dg-roles-practices-icon.png'
```
Comment (roadlittledawn) [inline src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-coe.mdx:21]: these may need to be separated by empty lines above and below

Comment (rhetoric101) [reply src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-coe.mdx:21]: @roadlittledawn, thanks for the line spacing tip!

**Comment by roadlittledawn** [comment]
the build error is happening when generating JSON (and converting MDX to HTML as part of it) at the end of the build.

Adding new lines between `import` statements may fix that (like [other docs do it](https://github.com/newrelic/docs-website/raw/develop/src/content/docs/new-relic-solutions/solve-common-issues/find-help-use-support-portal.mdx). If that doesn't fix it, converting them to markdown images would fix it

File: src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-coe.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-coe.mdx:21]: @roadlittledawn OK, I'll add line feeds.

**Comment by JimHagan** [comment]
> the build error is happening when generating JSON (and converting MDX to HTML as part of it) at the end of the build.
> 
> Adding new lines between `import` statements may fix that (like [other docs do it](https://github.com/newrelic/docs-website/raw/develop/src/content/docs/new-relic-solutions/solve-common-issues/find-help-use-support-portal.mdx)). If that doesn't fix it, converting them to markdown images would fix it

Looking better

**Review by rhetoric101 — APPROVED** [review]
@JimHagan, thanks for improving these OMA docs! These changes look good!

**Pull request merged**